### ISSUE TITLE
Elevated plasma Iso-/Butyrylcarnitine, C4:0

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5960]
**Preferred term label:** Elevated plasma Iso-/Butyrylcarnitine, C4:0

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Iso-/Butyrylcarnitine concentration in the blood., PMID: 30985723

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
